snapshots/overlay/overlay.go

```diff
@@ -67,13 +68,21 @@ func WithUpperdirLabel(config *SnapshotterConfig) error {
 	return nil
 }
 
+// WithMountOptions defines the default mount options used for the overlay mount.
+// NOTE: Options are not applied to bind mounts.
+func WithMountOptions(options []string) Opt {
+	return func(config *SnapshotterConfig) error {
+		config.mountOptions = options
```

Comment:
```suggestion
		config.mountOptions = append(config.mountOptions, options)
```
Is it better to use an append operation here?

Reply:
You are right, updated